### PR TITLE
Adding a Release Action in the build-release flow

### PR DESCRIPTION
**Pull request by itroyano**
Generate a release with notes using https://github.com/marvinpinto/action-automatic-releases.

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -37,3 +37,15 @@ jobs:
 
     - name: Print image url
       run: echo "Image pushed to ${{ steps.push-image.outputs.registry-paths }}"
+      
+    - name: GitHub Release
+      id: github-release
+      uses: "marvinpinto/action-automatic-releases@latest"
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
+        automatic_release_tag: "latest"
```
[the remark this answers is not in the document]
Comment (jomkz) [reply .github/workflows/build-release.yml:46]: ping @itroyano

**Comment by itroyano** [comment]
Fixes: #338

**Comment by jomkz** [comment]
Did we ever figure out what we are going to do with `automatic_release_tag`?

**Comment by itroyano** [comment]
I'm taking this back to the drawing board and will put up a detailed issue & design.

**Pull request closed without merging**